security-command-center/snippets/src/main/java/vtwo/notifications/DeleteNotification.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START securitycenter_delete_notification_config]
+
+package vtwo.notifications;
+
+import com.google.cloud.securitycenter.v2.DeleteNotificationConfigRequest;
+import com.google.cloud.securitycenter.v2.SecurityCenterClient;
+
+import java.io.IOException;
+
+public class DeleteNotification {
+    public static void main(String[] args) throws IOException {
+        // parentId: must be in one of the following formats:
+        //    "organizations/{organization_id}"
+        //    "projects/{project_id}"
+        //    "folders/{folder_id}"
+        String projectId = "{project-id}";
+        // Specify the location to list the findings.
+        String location = "global";
+        String notificationConfigId = "{your-notification-id}";
+
+        deleteNotificationConfig(projectId, location,notificationConfigId);
+    }
+    // Delete a notification config.
```

Comment:
add space above (although I think the space will be auto added once you install the java formatter)